src/main/python/inverse_parser/_attribute_setter.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Aniskov N.
+import ast
+import xml.etree.ElementTree as ET
+from pydoc import locate
+
+import src.main.python.inverse_parser._node_restorer as nr
+from src.main.python.inverse_parser._xml_node_children_getter import _XmlNodeChildrenGetter
+from src.main.util.const import DEFAULT_ENCODING
+
+
+class _AttributeSetter:
+
+    @staticmethod
+    def set_const_value(xml_node: ET.Element, py_node: ast.AST, py_node_attrib_name: str) -> None:
+        """
+        :param xml_node:
+        :param py_node:
+        :param py_node_attrib_name:
+        """
+        if 'value_type' not in xml_node.attrib:
+            raise RuntimeError(f'missing value_type attribute in {xml_node.tag} node')
```

Comment:
Please, in addition to raising an error, add a message to the logger about it. (for this case you should use `log_and_raise_error` function from the `util` module) Sometimes it is easier to read all logs to define what was wrong. It would be great if you used a logger everywhere to record intermediate information.